op-deployer/pkg/deployer/pipeline/alt_da.go

```diff
@@ -27,11 +27,16 @@ func DeployAltDA(env *Env, intent *state.Intent, st *state.State, chainID common
 		return nil
 	}
 
+	chainIndex, err := findChainIndex(st.Chains, chainID)
+	if err != nil {
+		return err
+	}
+
 	var dao opcm.DeployAltDAOutput
 	lgr.Info("deploying alt-da contracts")
 	dao, err = opcm.DeployAltDA(env.L1ScriptHost, opcm.DeployAltDAInput{
 		Salt:                     st.Create2Salt,
-		ProxyAdmin:               st.ImplementationsDeployment.OpcmProxyAddress,
+		ProxyAdmin:               st.Chains[chainIndex].ProxyAdminAddress, // TODO: @blmalone explain this change to slipper.
```

Comment:
@mslipper I think this was a bug. Let me know if this fix looks good to you.

Reply:
yes that looks like a bug, good catch